ssl/t1_lib.c

```diff
@@ -788,7 +792,8 @@ static const uint16_t tls_default_sigalg[] = {
     TLSEXT_SIGALG_ecdsa_sha1, /* SSL_PKEY_ECC */
     TLSEXT_SIGALG_gostr34102001_gostr3411, /* SSL_PKEY_GOST01 */
     TLSEXT_SIGALG_gostr34102012_256_gostr34112012_256, /* SSL_PKEY_GOST12_256 */
-    TLSEXT_SIGALG_gostr34102012_512_gostr34112012_512 /* SSL_PKEY_GOST12_512 */
+    TLSEXT_SIGALG_gostr34102012_512_gostr34112012_512, /* SSL_PKEY_GOST12_512 */
+    0 /* SSL_PKEY_ED25519 */
```

Comment:
The previous ECC curves could all be used for both ECDH and ECDSA. Ed25519 can be used for signing only and X25519 for ECDH: though there are similarities they're treated as distinct algorithms. The supported curves extension has an entry for x25519 but not Ed25519.

I'm not sure why that paragraph mentions TLS 1.1 and EdDSA. While it could theoretically be used there is no signalling mechanism for a client to indicate it supports Ed25519.